website/docs/getting-started/index.mdx

```diff
@@ -476,9 +459,9 @@ server.listen().then(({ url }) => {
 import { createServer } from '@graphql-yoga/node'
 import { readFileSync } from 'fs'
 
-import { Resolvers } from './resolvers-types';
+import { Resolvers } from './resolvers-types'
 
-const typeDefs = readFileSync('./schema.graphql').toString('utf-8')
+const typeDefs = readFileSync('./schema.graphql', 'utf8')
```

Comment:
`.toString('utf-8')` is redundant, `utf8` can be passed as 2nd argument to `readFileSync()`